detect_variants/vep.cwl

```diff
@@ -6,20 +6,18 @@ label: "Ensembl Variant Effect Predictor"
 baseCommand: ["/usr/bin/perl", "/usr/bin/variant_effect_predictor.pl"]
```

Comment:
Added. I reran the vep.cwl and the output VCF now contains the DownstreamProtein and WildtypeProtein fields.